docs/mutations.md

```diff
@@ -257,28 +257,25 @@ controls whether existing mutations are kept or discarded (the default is `keep=
 For instance, in final code block in {ref}`sec_mutations_time_span`, mutations were
 progressively added to a simulated tree sequence, beginninng with the oldest time period.
 
-While it is simple to add younger mutations to a tree sequence which already contains
-mutations, adding a mutation ancestral to an existing mutation will result in
-an error unless `add_ancestral=True` is specified, as in the following code block:
+While it is more natural to add younger mutations to a tree sequence which already contains
+mutations, you can also add mutations ancestral to an existing mutations,
```

Comment:
There's a typo here. "an" disagrees with "mutations". I am not sure whether "an existing mutations" should be plural or singular. @petrelharp?

Reply:
It's a typo all right. I've created an issue #1641